test/unit/util/mapKeysTest.ts

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from 'chai';
```

Comment:
Are there no other unit tests for the other scratch org creation logic? There's a lot of untested code

Reply:
Yes, that's true, tons of untested code in fact, there are portions that can be `ported` from toolbelt but my best guess is that 80% of them will have to be re-written, I created this W-9991956 to tackle this problem, though I will probably put all changes here in this PR